test_scripts/RC/Capabilities/005_Resend_only_supported_parameters.lua

```diff
@@ -23,26 +23,29 @@ local available_params =
     moduleType = "RADIO",
     radioControlData = {frequencyInteger = 1, frequencyFraction = 2, band = "AM"}
 }
-local absent_params = {moduleType = "RADIO", radioControlData = {frequencyInteger = 1, frequencyFraction = 2}}
+local absent_params = {moduleType = "RADIO", radioControlData = {signalStrength = 60}}
 
 --[[ Local Functions ]]
-local function setVehicleData(params, self)
-	local cid = self.mobileSession:SendRPC("SetInteriorVehicleData", {moduleData = params})
+local function rpcIsAllowed(params, self)
+    local cid = self.mobileSession:SendRPC("SetInteriorVehicleData", {moduleData = params})
 
-	if params.radioControlData.frequencyInteger then
-		EXPECT_HMICALL("RC.SetInteriorVehicleData",	{
-            appID = commonRC.getHMIAppId(1),
-			moduleData = params})
-		:Do(function(_, data)
-				self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {
-					moduleData = params})
-			end)
-		self.mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
-	else
-		EXPECT_HMICALL("RC.SetInteriorVehicleData"):Times(0)
-		self.mobileSession:ExpectResponse(cid, { success = false, resultCode = "UNSUPPORTED_RESOURCE" })
-        common_functions.DelayedExp(commonRC.timeout)
-	end
+    EXPECT_HMICALL("RC.SetInteriorVehicleData",	{
+        appID = commonRC.getHMIAppId(1),
+        moduleData = params})
+    :Do(function(_, data)
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {
+            moduleData = params})
+    end)
+    self.mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[ Local Functions ]]
```

Comment:
@Itileda Section "Local functions" is already defined at line 28. Please remove current one.